NEWS.md

```diff
@@ -7,6 +7,8 @@ argument in `derive_extreme_event()`. This allows to define events based on more
 than one observation, e.g., events which need to be confirmed by a second
 assessment. (#1960)
 
+- `data/atoxgr_criteria_daids.rda` added, which holds metadata for [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
+
```

Comment:
So the issue for this is pretty bland...but I think the discussion in the PR is very interesting.  Shall we link the PR number here?

Reply:
we should add the link to the PR under the new argument `signif_dig` for `derive_var_atoxgr_dir`, right? As the significant digits was the main discussion point in thre PR?